### ISSUE TITLE
Missing sounds?

### ISSUE THREAD

**Issue by YanikaZerafa** [Pantalleresco/IMTAssignment2016#13]
Certain powerups don't have sound effects when i trigger them, will they be implemented to the game?

**Comment by Pantalleresco** [comment]
Yes they will implemented in the future, i will be making a list of all the powerups and their designated sounds.
